ignite/cmd/plugin.go

```diff
@@ -55,6 +68,16 @@ func parseLocalPlugins(rootCmd *cobra.Command) (cfg *pluginsconfig.Config, path
 	return cfg, pluginsPath, err
 }
 
+func parseGlobalConfig() (cfg *pluginsconfig.Config, path string, err error) {
+	globalPath, err := plugin.PluginsPath()
+	if err != nil {
+		return cfg, globalPath, err
+	}
+
+	cfg, err = pluginsconfig.ParseFile(globalPath)
```

Comment:
I have doubt that this works well since `globalPath` is equal to `$HOME/.ignite/plugins` and not `$HOME/.ignite/plugins/plugins.yml`. That say, this works with my refac where I changed `ParseFile` to `ParseDir` ;)